templates/vue/src/components/NodeModal.vue

```diff
@@ -104,6 +110,7 @@
               <b-form-input
                 id="node-h5p-media-duration"
                 v-model="node.mediaDuration"
+                data-testid="node-h5pDuration"
```

Comment:
Can you not use the id attribute instead of the test-specific data attribute? Or the class attribute if it's repeatable? Would be much cleaner.

Reply:
Hey @narendrasss still waiting for a response here.

Reply:
Oops, just saw your response - ignore.